deeplearning4j/deeplearning4j-nlp-parent/deeplearning4j-nlp/src/main/resources/sentiment/sentiwordnet.txt

```diff
@@ -79810,7 +79810,7 @@ n	11348160	0	0	tree#3 sir_herbert_beerbohm_tree#1	English actor and theatrical p
 n	11348356	0	0	trevelyan#2 sir_george_otto_trevelyan#1 george_otto_trevelyan#1	English historian who wrote a history of the American revolution and a biography of his uncle Lord Macaulay (1838-1928)
 n	11348584	0	0	trevelyan#1 george_macaulay_trevelyan#1	English historian and son of Sir George Otto Trevelyan whose works include a social history of England and a biography of Garibaldi (1876-1962)
 n	11348812	0	0	trevino#1 supermex#1 lee_trevino#1 lee_buck_trevino#1	United States golfer (born in 1939)
-n	11348946	0	0	trevithick#1 richard_trevithick#1	English engineer who built the first railway locomotive (1771-1833)
+n	11348946	0	0	trewithick#1 richard_trewithick#1	English engineer who built the first railway locomotive (1771-1833)
```

Comment:
This needs to be reverted too. I thought at marking just one instance in this file would have been enough.